faqs/galaxy/galaxy_creating_an_account.md

```diff
@@ -0,0 +1,17 @@
+---
+title: How to create an account on a public Galaxy instance?
+area: Account
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+1. To create an account at any public Galaxy instance, choose your server from the available [list of Galaxy Platform](https://galaxyproject.org/use/).
+ There are 3 public  Galaxy servers, which are [UseGalaxy.eu](https://usegalaxy.eu/), [UseGalaxy.org (Main)](https://usegalaxy.org/) and [UseGalaxy.org.au](https://usegalaxy.org.au/).
+
+3. Click on "Login or Register" in the masthead on the server. When logged in, a User menu becomes available that is specific to your account.
+
+4. Please make sure your registered account email address is valid so you can confirm your new account, and our administrative support team can contact/reply to you when needed. Account email addresses are case sensitive. 
```

Comment:
What about changing the phrasing of this step so that it becomes an action? For example: `Check your email and confirm your new account...` we could also merge it with 5. As it is now, number 4 is more a "note" than an action and we could put it at the end.

Reply:
Yes I think it will precise and straight to the point. Kindly have a look at the changes we made.